pymess/backend/dialer/daktela.py

```diff
@@ -98,7 +96,7 @@ def _update_message_with_error(self, message, error_message, is_final_state=True
         )
         message_kwargs = {
             'state': DialerMessage.STATE.ERROR,
-            'error': ', '.join(error_message) if isinstance(error_message, Sequence) else str(error_message),
+            'error': ', '.join(error_message) if isinstance(error_message, list) else str(error_message),
```

Comment:
Wouldn't be better to just test string?
```suggestion
            'error': ', '.join(error_message) if not isinstance(error_message, str) else str(error_message),
```

Reply:
IMO currently with default conversion to `str` it is less error prone as objects have usually `__str__` implemented. error_message could be str, list or Exception so it would fail then...

Reply:
OK, agreed.